src/main/java/domain/player/Player.java

```diff
@@ -0,0 +1,46 @@
+package domain.player;
+
+
+import domain.Direction;
+
+public class Player {
+    private final PlayerName name;
+    private final PlayerPosition position;
+
+    public Player(String playerName, int playerPosition) {
+        this.name = new PlayerName(playerName);
+        this.position = new PlayerPosition(playerPosition);
+    }
+
+    public void playerMove(Direction direction) {
+        if (direction.equals(Direction.RIGHT)) {
+            position.moveRight();
+        }
+        if (direction.equals(Direction.LEFT)) {
+            position.moveLeft();
+        }
+    }
```

Comment:
```suggestion
    public void move(Direction direction) {
        if (direction.equals(Direction.RIGHT)) {
            position.moveRight();
        }
        if (direction.equals(Direction.LEFT)) {
            position.moveLeft();
        }
    }
```

player.move(direction) 이면 충분하지 않을까요?